### ISSUE TITLE
Execute a script from inside a script

### ISSUE THREAD

**Issue by dzach** [openpnp/openpnp#523]
# Feature Request
## Describe the Feature
Run scripts (any accepted language) from inside another script.

## How Will It Look?
NA, but it shouldn't give this error:
![scripting](https://cloud.githubusercontent.com/assets/1109829/25066727/7d5bd636-2237-11e7-9cbe-49d37efc0652.png)

## Provide Examples
Should be able to run:
```
scripting.execute(new File(scripting.getScriptsDirectory().getAbsolutePath()+"/tests/hi.js"));
```
or similar, from inside another script, in any of the existing engines.

## Details
Is there a reason why `scripting.execute`, i.e.:
```
private void execute(File script)
```
is `private` instead of `public`?

Note: `scripting.on()` is `public` and allows the execution of scripts in the Events subdir. 

I have lightly tested it and it doesn't seem to cause any problem. I can create a pull request with this change, if that's ok.

**Comment by vonnieda** [comment]
I'm fine with the change from private to public. Go ahead and change both execute() methods.

**Comment by vonnieda** [comment]
Fixed in #524
